### DISCUSSION TITLE
Ordered lists

### DISCUSSION THREAD

**Discussion by borg** [keystonejs/keystone#6579] [Event Q&A Q3 2021]
Computers seem quite good at this sort of thing, why is it seemingly such a challenge for the KS approach? Clients expect being able to order items in  a custom way. (I know it's on the road map so you can ignore if it's around the corner)

**Comment by MurzNN** [comment]
Currently there is limitation of Prisma database driver: https://github.com/prisma/prisma/issues/1955 but we can reuse some field like "weight" in target entity to store the needed order.

Also we can reuse the JSON array (or postgresql "array of strings" field) to store ordered list as separate way than relations.

**Comment by raveling** [comment]
Thanks for your question @borg 🙌 
We answered this in the recent community Q&A event:
https://www.youtube.com/watch?v=r1IJh-iHm1c&t=1811s
